src/ml/neural_net/mps_graph_trainer.mm

```diff
@@ -63,23 +66,27 @@ int TCMPSDeleteGraphModule(MPSHandle handle) {
 EXPORT int TCMPSTrainStyleTransferGraph(MPSHandle handle, int index, TCMPSFloatArrayRef inputs,
                                         TCMPSFloatArrayRef labels, TCMPSFloatArrayRef* loss_out) {
   API_BEGIN();
-
-  style_transfer *obj = reinterpret_cast<style_transfer *>(handle);
-  float_array* inputs_ptr = reinterpret_cast<float_array*>(inputs);
-  float_array* labels_ptr = reinterpret_cast<float_array*>(labels);
-
-  shared_float_array inputs_array(
-      std::make_shared<external_float_array>(*inputs_ptr));
-  shared_float_array labels_array(
-      std::make_shared<external_float_array>(*labels_ptr));
-  shared_float_array index_array(
-      std::make_shared<float_scalar>(index));
-
-  auto outputs = obj->train({ { "input",  inputs_array },
-                              { "labels", labels_array },
-                              { "index",  index_array  } });
-  shared_float_array* loss = new shared_float_array(outputs.at("loss"));
-  *loss_out = reinterpret_cast<TCMPSFloatArrayRef>(loss);
+  #ifdef HAS_MACOS_10_15
+    style_transfer *obj = reinterpret_cast<style_transfer *>(handle);
```

Comment:
Don't change the indent level of the code.

The preprocessor direction should be at zero indent

https://google.github.io/styleguide/cppguide.html#Preprocessor_Directives

Consider just using clang-format

Reply:
Used clang-format on the file!